app/assets/javascripts/jquery_overrides.js

```diff
@@ -37,10 +37,20 @@ $.ajaxSetup({
     }),
     "text script": logError(function (text) {
```

Comment:
.. because `logError` is a terrible name for a function that logs errors :trollface: